nested-spire/README.md

```diff
@@ -0,0 +1,226 @@
+
+# Overview
+
+Nested SPIRE allows SPIRE Servers to be “chained” together, and for all SPIRE Servers to issue identities in the same trust domain, meaning all workloads identified in the same trust domain are issued identity documents that can be verified against the root keys of the trust domain.
+
+Nested topologies work by co-locating a SPIRE Agent with every downstream SPIRE Server being “chained”. The downstream SPIRE Server obtains credentials over the Workload API that it uses to directly authenticate with the upstream SPIRE Server to obtain an intermediate CA.
+
+To demonstrate a deployment of SPIRE in a nested topology we create a scenario using Docker Compose with a root SPIRE deployment and two SPIRE deployments nested to it.
+
+![Nested SPIRE diagram][nested-SPIRE-diagram]
+
+[nested-SPIRE-diagram]: images/Nested_SPIRE_Diagram.png "Nested SPIRE Deployment diagram"
```

Comment:
this diagram is not right, it is true that nestedA uses agent but it uses agent in order to fetch an svid for itself and then uses that svid to communicate with root-server in order to push mints, maybe you can udpate that diagram to reflect that?